x-pack/legacy/plugins/siem/public/containers/case/use_get_cases.tsx

```diff
@@ -142,5 +191,55 @@ export const useGetCases = (): [
       didCancel = true;
     };
   }, [state.queryParams, state.filterOptions]);
-  return [state, setQueryParams, setFilters];
+  useEffect(() => fetchCases(), [state.queryParams, state.filterOptions]);
+  useEffect(() => {
+    if (doUpdate) {
+      fetchCases();
+      getCaseCount('open');
+      getCaseCount('closed');
+      setDoUpdate(false);
+    }
+  }, [doUpdate]);
+
+  const getCaseCount = async (caseState: keyof CaseCount) => {
```

Comment:
I talked to @stephmilovic and we agreed that the API have to get smarter, so we won't need to do this